geometry_msgs/Point[] visited_waypoints
---
bool responce